examples/astro/src/react/content-types/banner.tsx

```diff
@@ -0,0 +1,38 @@
+import useImageSrc from "../hooks/useImageSrc";
+
+import type { FC } from "react";
+import type { DotCMSContentlet } from "../../types";
+
+export type BannerProps = DotCMSContentlet;
+
+export const Banner: FC<BannerProps> = ({
```

Comment:
`FC` is the right choice here? I am always confused.

Reply:
There is another way to do it, that is more explicit, but `FC` is fine for an example